Configuration/ConsulRx.Configuration.csproj

```diff
@@ -1,26 +1,25 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <Version>1.1.0</Version>
+    <Version>2.0.0</Version>
     <Description>Provides a configuration source that pulls values from Consul's Service Catalog and KV 
     Store.</Description>
-    <Copyright>2017-2020</Copyright>
+    <Copyright>2017-2024</Copyright>
     <Authors>Andy Alm</Authors>
     <PackageTags>consul;configuration;reactive;extensions</PackageTags>
     <PackageProjectUrl>https://github.com/andyalm/consul-rx</PackageProjectUrl>
-    <PackageLicenseUrl>http://opensource.org/licenses/MIT</PackageLicenseUrl>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
     <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
     <PackageReleaseNotes>
-      - Removed file system caching by default as it does not play well in a serverless environment and is not really useful in a containerized one either. It can still be opted into.
-      - Started targeting netstandard2.0.
-      - Added a IConfigTypeConverter, which can be used to customize how a consul key is mapped to the config store.
+      - Update dependencies and target framework
     </PackageReleaseNotes>
     <LangVersion>Latest</LangVersion>
   </PropertyGroup>
   <ItemGroup>
     <ProjectReference Include="..\ConsulRx\ConsulRx.csproj" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Extensions.Configuration" Version="2.0.*" />
+    <PackageReference Include="Microsoft.Extensions.Configuration" Version="9.0.*" />
```

Comment:
this is too new. It will make it incompatible with .net 8 apps.

Reply:
generally, yes, a 9.x package will only support a 9.x runtime --

this is an outlier though, it multi-targets and offers broad compatibility
<img width="896" alt="image" src="https://github.com/user-attachments/assets/3ce5444c-b88f-47e0-bb10-3019b6ff003d">

Reply:
I still don't really want to force version 9 on every consumer. That package generally comes with the `Microsoft.AspNetCore.App` framework that ships with the runtime, and its a bit surprising to be bringing in version 9 on older .net versions if it wasn't explicitly asked for.

Reply:
that's fair.  looks like the minimum version on nuget.org that is still supported is 6.x.  seems like the sweet spot.